src/content/controller_content.ts

```diff
@@ -98,13 +99,6 @@ function* ParserController() {
                     response,
                 )
 
-		// show current keyEvents as a suffix of the contentState
```

Comment:
It was a pretty easy fix in the end - just had to move this block down until the parser returns the keys which could be part of a valid bind.